src/containers/Search/index.tsx

```diff
@@ -207,6 +210,9 @@ const Search = (homeProps: IHomeProps) => {
       document.removeEventListener('contextmenu', event => event.preventDefault())
     }
   }, [])
+  const windowWidth = window.innerWidth
+  const elementWidth = `calc(${windowWidth}px - 50%)`
```

Comment:
Don't calculate width like this. Give the search bar the same size as the form in smaller screen.